docs/deployment/migration.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Migration
+parent: Deployment
+nav_order: 7
+---
+
+# Migrating from Google Santa to NPS Santa
+
+## Overview
+This guide outlines the migration process from Google Santa to NPS Santa, designed to ensure a smooth transition with minimal security coverage gaps.
+
+If you are not currently running Google Santa, you can skip this doc and go straight to [Getting Started](getting-started.md).
+
+## Prerequisites
+- Active Google Santa installation
+- MDM (Mobile Device Management) - If you do not use an MDM jump to [2. Install NPS Santa](#2.-Install-NPS-Santa)
+- NPS Santa installer package
```

Comment:
Should this be a link to github.com/northpolesec/santa/latest?

Reply:
I was meant to be a link to the 2nd step of the migration doc. I fixed the link.